### DISCUSSION TITLE
Ghelper auto boot second screen

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#3277] [General]
I have a Rog Duo, has 2 screens. How do I get ghelper to auto open onto the second screen not the first? When I used armoury crate you could use screenxprt " which I don't use anymore" to pick which screen for certain programs to boot up onto.

**Comment by seerge** [comment]
@Reddot911 it opens on the main screen, next to the tray icon :) It's by design.

**Reply by ghost** [reply]
Was just curious if there was a way.
